app/views/start.njk

```diff
@@ -6,23 +6,25 @@
   {{ govukBreadcrumbs({
     items: [{
       text: "Home",
-      href: "#"
-    }, {
-      text: "Education, training and skills",
-      href: "#"
-    }, {
-      text: "Teaching and leadership",
-      href: "#"
-    }, {
-      text: "Teacher training and professional development",
-      href: "#"
-    }, {
-      text: "Initial Teacher Training (ITT)",
-      href: "#"
+      href: "/start"
```

Comment:
I think we want to remove this as it links to the current page.

Reply:
@Nitemaeric do you mean we remove all breadcrumbs from the start page?